cleaned_data/modsbyPID/rftaart:1.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mods xmlns="http://www.loc.gov/mods/v3"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:xlink="http://www.w3.org/1999/xlink"
+      xmlns:xs="http://www.w3.org/2001/XMLSchema"
+      xsi:schemaLocation="http://www.loc.gov/mods/v3 http://www.loc.gov/standards/mods/v3/mods-3-5.xsd">
+   <identifier type="local">RFTA_Braddock_001</identifier>
+   <identifier type="pid">rftaart:1</identifier>
+   <titleInfo>
+      <title>The home of Stacey Adam and Chris Szaton is still standing</title>
+   </titleInfo>
+   <originInfo>
+      <dateIssued>2021</dateIssued>
+      <dateIssued encoding="edtf" keyDate="yes">2021</dateIssued>
+      <publisher/>
+      <place>
+         <placeTerm valueURI="http://id.loc.gov/authorities/names/n79109786">Knoxville (Tenn.)</placeTerm>
+      </place>
+   </originInfo>
+   <abstract>Stacey Adam and Chris Szaton evacuated and were initially told that their house had burned. But when they were finally allowed to go back in, they realized their home was still standing. The burned-out foundations of their neighbor’s houses surrounded it. They described the destruction as something similar to a war zone.</abstract>
+   <name authority="naf"
+         valueURI="http://id.loc.gov/authorities/names/n2002022941">
+      <namePart>Braddock, Paige</namePart>
+      <role>
+         <roleTerm authority="marcrelator"
+                   valueURI="http://id.loc.gov/vocabulary/relators/art">Artist</roleTerm>
+      </role>
+   </name>
+   <name>
+      <namePart>Adam, Stacey</namePart>
+      <role>
+         <roleTerm authority="marcrelator"
+                   valueURI="http://id.loc.gov/vocabulary/relators/ive">Interviewee</roleTerm>
+      </role>
+   </name>
+   <name>
+      <namePart>Szaton, Chris</namePart>
+      <role>
+         <roleTerm authority="marcrelator"
+                   valueURI="http://id.loc.gov/vocabulary/relators/ive">Interviewee</roleTerm>
+      </role>
+   </name>
+   <physicalDescription>
+      <form authority="aat" valueURI="http://vocab.getty.edu/aat/300033973">drawings (visual works)</form>
+   </physicalDescription>
+   <subject authority="lcsh"
+            valueURI="http://id.loc.gov/authorities/subjects/sh85146723">
+      <topic>Wildfires</topic>
+   </subject>
+   <subject authority="lcsh"
+            valueURI="http://id.loc.gov/vocabulary/graphicMaterials/tgm003413">
+      <topic>Dwellings</topic>
+   </subject>
+   <typeOfResource>text</typeOfResource>
+   <relatedItem displayLabel="Project" type="host">
+      <titleInfo>
+         <title>Rising from the Ashes: The Chimney Tops 2 Wildfires in Memory and Art.</title>
+      </titleInfo>
+   </relatedItem>
+   <relatedItem type="references">
+      <location>
+         <url>Waiting on interview transcript</url>
```

Comment:
@logantillman - Here is another item that we still need the link for. @markpbaggett - can you comment on this PR with any links that do become available once you add the captions/content?